libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -818,7 +818,7 @@ string IRGenerator::dispatchRoutine(ContractDefinition const& _contract)
 				return(memPos, sub(memEnd, memPos))
 			}
 			</cases>
-			default {}
+			default { }
```

Comment:
Why is this change needed?

Reply:
Not needed, included by mistake, removing it now